docs/source/nlp/text_normalization.rst

```diff
@@ -17,18 +17,38 @@ system consists of two individual components:
 The typical workflow is to first train a DuplexTaggerModel and also a DuplexDecoderModel. An example training script
 is provided: `duplex_text_normalization_train.py <https://github.com/NVIDIA/NeMo/blob/main/examples/nlp/duplex_text_normalization/duplex_text_normalization_train.py>`__.
 After that, the two trained models can be used to initialize a `DuplexTextNormalizationModel <https://github.com/NVIDIA/NeMo/blob/main/nemo/collections/nlp/models/duplex_text_normalization/duplex_tn.py/>`__ that can be used for end-to-end inference.
-An example script for evaluation and inference is provided: `duplex_text_normalization_test.py <https://github.com/NVIDIA/NeMo/blob/main/examples/nlp/duplex_text_normalization/duplex_text_normalization_test.py>`__. The term
-*duplex* refers to the fact that our system can be trained to do both TN and ITN. However, you can also specifically train the system for only one of the tasks.
+An example script for evaluation is provided here: `duplex_text_normalization_test.py <https://github.com/NVIDIA/NeMo/blob/main/examples/nlp/duplex_text_normalization/duplex_text_normalization_test.py>`__. 
+The script for inference of the full pipeline is provided here: `duplex_text_normalization_infer.py <https://github.com/NVIDIA/NeMo/blob/main/examples/nlp/duplex_text_normalization/duplex_text_normalization_infer.py>`__. 
+This script also allows you to interactively run inference in the terminal. 
+The term *duplex* refers to the fact that our system can be trained to do both TN and ITN. However, you can also specifically train the system for only one of the tasks.
 
-NeMo Data Format
+
+Quick Start Guide
+-----------------
+
+To run the pretrained models interactively see :ref:`inference_text_normalization`.
+
+Available models
+^^^^^^^^^^^^^^^^
+
+.. list-table:: *Pretrained Models*
+   :widths: 5 10
+   :header-rows: 1
+
+   * - Model
+     - Pretrained Checkpoint
+   * - neural_text_normalization_t5
+     - https://ngc.nvidia.com/catalog/models/nvidia:nemo:neural_text_normalization_t5
```

Comment:
Is this chpt compatible with all the changes?

Reply:
no, i will update the new ckpts and this documentation